README.md

```diff
@@ -13,7 +13,7 @@ This repository is a live example of a simple custom TensorBoard, which implemen
 
 To test out the custom “greeter TensorBoard” with the Greeter Plugin, first [install Bazel] and [install TensorFlow].
 
-> **Note:** This example requires TensorFlow 1.3 to run. As of 2017-07-24, TensorFlow 1.3 isn’t out yet, which means you need to install one of TensorFlow’s cutting edge nightly binaries. You can find the nightly download urls [here](https://github.com/tensorflow/tensorflow/blob/master/README.md).
+> **Note:** This example requires TensorFlow 1.3 to run. Moreover, please use Bazel version 0.19.2 as 0.20.0 made some backwards incompatible change and their rules are not updated accordingly yet.
```

Comment:
Awesome! While updating this, would you mind changing `This example requires TensorFlow 1.3 to run` to `This example requires the latest TensorFlow to run`? It looks like a good idea to be tracking the latest TF. Thanks!

Reply:
Sure, that's a good idea!